LICENSE-MIT

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2018 Ashley Williams <ashley666ashley@gmail.com>
+Copyright (c) 2020 Cloudflare, Inc. <wrangler@cloudflare.com>
```

Comment:
now this PR does two things, idk how  licensing things  work

Reply:
uhh woops